backend/components/src/test/java/org/eclipse/sw360/components/db/BulkDeleteUtilTest.java

```diff
@@ -517,6 +521,60 @@ public void testDeleteBulkRelease002() throws Exception {
         }
     }
     
+    @Test
+    public void testDeleteBulkRelease_ExternalLink001() throws Exception {
+        if (!isFeatureEnable()) {
+            System.out.println("BulkReleaseDeletion is disabled. these test is Skipped.");
```

Comment:
It will be good not to keep `System.out.println("-------")` statement instead we can put a logger message.

Reply:
Thank you for the comments!
Changed to logger and resolved the merge conflict.